src/pybind/mgr/dashboard/frontend/src/app/shared/pipes/list-to-html.pipe.ts

```diff
@@ -0,0 +1,55 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+import { RelativeDatePipe } from './relative-date.pipe';
+import { TextToLabelPipe } from './text-to-label.pipe';
+
+@Pipe({
+  name: 'listToHtml'
+})
+export class ListToHtmlPipe implements PipeTransform {
+  level: string;
+  levelIcon: string;
+  oldDate: any;
+  transformDate: any;
+  relativeDate: string;
+  subject: string;
+  labeledSubject: string;
+  message: string;
+  indentedMessage: string;
+  newEvent: Array<any> = [];
+
+  transform(value: Array<any>): string {
+    const pipe = new RelativeDatePipe();
```

Comment:
Why are these pipes not instantiated using the constructor (Dependency Injection?)

Reply:
Thanks @alfonsomthd . Made this change now.